websites/V/vscode.dev/README.md

```diff
@@ -0,0 +1,29 @@
+# How to custom string
+Custom string for the details or state <br>
+This is a string that can be used (there must be a `%` sign before and after every time) <br>
+
+- `%file%` will be replaced with the current file name.
+- `%branch%` will be replaced with the current branch name. (remote URL only)
+- `%error%` will be replaced with error current workspace folder.
+- `%problems%` will be replaced with problems current workspace folder.
+- `%workspace%` will be replaced with the current workspace name.
+- `%lang%` will be replaced with the language current file.
+- `%encoding%` will be replaced with the encoding.
+- `%selection%` will be replaced with the current line.
```

Comment:
I would use a [markdown table](https://www.tablesgenerator.com/markdown_tables) for this with the headers "Parameter", "Description" and "Example"